tests/utils.py

```diff
@@ -16,11 +15,7 @@
 async def run_server(config: Config, sockets=None):
     server = Server(config=config)
     cancel_handle = asyncio.ensure_future(server.serve(sockets=sockets))
-
-    # Workaround for Python 3.9.7 (see https://bugs.python.org/issue45097)
-    with warnings.catch_warnings():
-        warnings.filterwarnings("ignore", category=DeprecationWarning)
-        await asyncio.sleep(0.1)
```

Comment:
this context manager applies the warning filter to all other tasks (and threads) running concurrently

Reply:
see https://bugs.python.org/issue37604